docs/02-app/01-building-your-application/02-data-fetching/03-forms-and-mutations.mdx

```diff
@@ -374,7 +383,9 @@ export function SubmitButton() {
   const { pending } = useFormStatus()
 
   return (
-    <button disabled={pending}>{pending ? 'Submitting...' : 'Submit'}</button>
+    <button type="submit" aria-disabled={pending}>
```

Comment:
Prefer `aria-disabled` instead of `disabled`, as some screen readers will hide the `button` element when `disabled` is `true`.